theme/css/swiper.css

```diff
@@ -0,0 +1,50 @@
+/*testimonial */
+.testimonial-container.swiper-container {
+  width: 100%;
+  height: 450px;
+  padding-top: 10px;
+}
+
+.testimonial-container.swiper-container.full-width {
+  width: 100vw;
+  margin-left: -50vw;
+  margin-right: -50vw;
+}
+
+.testimonial-container.swiper-container .swiper-wrapper {
+  width: 100%;
+}
+
+.testimonial-container .swiper-slide {
+  text-align: center;
+  font-size: 18px;
+  background: #fff;
+  width: 90%;
+  height: 400px;
+  /* Center slide text vertically */
+  display: -webkit-box;
+  display: -ms-flexbox;
+  display: -webkit-flex;
+  display: flex;
+  -webkit-box-pack: center;
+  -ms-flex-pack: center;
+  -webkit-justify-content: center;
+  justify-content: center;
+  -webkit-box-align: center;
+  -ms-flex-align: center;
+  -webkit-align-items: center;
+  align-items: center;
+  border-radius: 10px;
+  transition: all .4s ease-in-out;
+  box-shadow: 0 4px 8px 0 rgba(0,0,0,0.2);
+  padding: 5px 20px;
+}
+
+.swiper-slide:hover {
```

Comment:
This needs to be more specific, include the testimonial container class.

Reply:
done